.ci/azure-pipelines/build_linux.bash

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -ex
 #
-# Copyright 2019-2021 The Mumble Developers. All rights reserved.
+# Copyright 2020-2021 The Mumble Developers. All rights reserved.
```

Comment:
This changes the source year rather than only updating new files?

Reply:
Yes - that's what the old version of the script messed up and the new one fixes